doc/HD44780.rst

```diff
@@ -63,15 +69,15 @@ ignores.
   below the lowest line on the device will not be displayed.
 
 Graphics-mode Usage
-^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~
 It is also possible to display graphical content using the `display` method
-which works similarly to other luma.lcd and luma.oled displays.  However there is
-a significant limitation.  There can only be 8 characters worth of 'custom'
+which works similarly to other ``luma.lcd`` and ``luma.oled`` displays.  However
+there is a significant limitation.  There can only be 8 characters worth of 'custom'
```

Comment:
Curious to know what is the behaviour if you exceed the 8 characters worth of custom content? Does it get garbled output, empty output or a runtime error?

Reply:
If you exceed 8 characters, all characters that do not match an existing character in the font table of what is in the custom space are displayed as the undefined character which can be changed during initialization of the hd44780 class.  The default value is '_'.  

I'll see if I can make that clearer in the documentation.